Makefile

```diff
@@ -174,7 +173,7 @@ ifeq ($(dockerized),yes)
 			make flutter.clean dockerized=no
 else
 	flutter clean
-	rm -rf .cache/pub/ doc/ \
+	rm -rf .cache/pub/ doc/ symbols/ \
```

Comment:
@SleepySquash shoudn't it be `debug/` according to renewed `.gitignore` file?

Reply:
@tyranron, oopsie, you're right, indeed it should.